### PR TITLE
Add update loop

### PR DESCRIPTION
**Pull request by pparkkin**
I felt like the starter could do with an update loop. Feels like something you almost always need anyway.

### CODE REVIEW

**Comment by GuillaumeDesforges** [comment]
Hi! Thanks for the PR 🙂 
You are right, an update loop is most needed in this starter!

That being said, I'm against promoting the use of wild global variables in _any_ code, so I wouldn't like to be written in a repo I own.
I'm being picky here, but I'd really like if we could find another pattern, so as to invite people to start on "solid foundations".

If we can't think of any better way to store the global state, we can move forward with what you've written.

File: src/index.ts [hunk 1]
```diff
@@ -37,6 +37,24 @@ const main = async () => {
     });
 
     document.body.appendChild(app.view);
+
+    app.ticker.add(update, sprite);
+};
+
+
+var xv = 1;
```
Comment (GuillaumeDesforges) [inline src/index.ts:45]: Wild global variables

**Review by GuillaumeDesforges — CHANGES_REQUESTED** [review]
Hi! Thanks for the PR 🙂 
You are right, an update loop is most needed in this starter!

That being said, I'm against promoting the use of wild global variables in _any_ code, so I wouldn't like to be written in a repo I own.
I'm being picky here, but I'd really like if we could find another pattern, so as to invite people to start on "solid foundations".

If we can't think of any better way to store the global state, we can move forward with what you've written.

**Comment by pparkkin** [comment]
You're right about the global variables of course. I pushed a commit that I hope fixes it.

**Review by GuillaumeDesforges — APPROVED** [review]

**Comment by GuillaumeDesforges** [comment]
Awesome job, thank you very much for your contribution!

**Pull request merged**